en/mkdocs.yml

```
@@ -123,6 +123,7 @@ nav:
       - Common Parallel Strategies: parallelism/01_introduction.md
       - Global View: parallelism/02_sbp.md
       - Global Tensor: parallelism/03_consistent_tensor.md
+      - 2D SBP: parallelism/07_2dsbp.md
```

```suggestion
      - 2D SBP: parallelism/04_2d-sbp.md
```

和中文版本的文件名一致吧，另外那个文件的文件名也相应修改。